docs/src/appendix/deprecated-features.md

```diff
@@ -31,7 +17,7 @@ exporters:
 
 In terms of specifying values, there were two minor changes:
 - Memory sizes are now specified like this: `512MB` (old way: `512M`)
-- Durations, e.g. timeouts, can now also be given in ISO-8601 Durations format. However you can still use the established way and specify a timeout of `30s`  
+- Durations, e.g. timeouts, can now also be given in ISO-8601 Durations format. However you can still use the established way and specify a timeout of `30s`
 
 ### Legacy Environment Variables
```

Comment:
This section needs to be removed. The legacy environment variables were mapped in the TOML configuration module.

Reply:
Done :)